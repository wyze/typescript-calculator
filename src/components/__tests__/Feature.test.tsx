import { h } from 'preact'
import Feature, { Emoji } from '../Feature'
import snapshot from './utils/snapshot'

const props = {
  emoji: Emoji.Checkmark,
  text: 'Some Feature',
}

const propsWithSoon = {
  ...props,
  emoji: Emoji.Soon,
}

describe('<Feature />', () => {
  it('renders', () => snapshot(<Feature {...props} />))
  it('renders with Soon emoji', () => snapshot(<Feature {...propsWithSoon} />))
})
