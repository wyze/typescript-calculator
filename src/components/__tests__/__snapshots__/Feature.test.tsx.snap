// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Feature /> renders 1`] = `
.glamor-0,
[data-glamor-0] {
  align-items: center;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-basis: 40%;
  height: 2em;
  -webkit-box-align: center;
  -webkit-align-items: center;
  -webkit-flex-basis: 40%;
}

@media only screen and (min-width: 53.125em) {
  .glamor-0,
  [data-glamor-0] {
    flex-basis: 35%;
    -webkit-flex-basis: 35%;
  }
}

<div
  className="glamor-0"
>
  <Checkmark />
  <h3>
    Some Feature
  </h3>
</div>
`;

exports[`<Feature /> renders with Soon emoji 1`] = `
.glamor-0,
[data-glamor-0] {
  align-items: center;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-basis: 40%;
  height: 2em;
  -webkit-box-align: center;
  -webkit-align-items: center;
  -webkit-flex-basis: 40%;
}

@media only screen and (min-width: 53.125em) {
  .glamor-0,
  [data-glamor-0] {
    flex-basis: 35%;
    -webkit-flex-basis: 35%;
  }
}

<div
  className="glamor-0"
>
  <Soon />
  <h3>
    Some Feature
  </h3>
</div>
`;
